src/store/getters/state-getters.ts

```diff
@@ -274,12 +273,12 @@ export const getNameRequest = (state: StateIF): NameRequestIF => {
 
 /** The Name Request Number. */
 export const getNameRequestNumber = (state: StateIF): string => {
-  return getNameRequest(state).nrNumber
+  return getNameRequest(state)?.nrNumber
```

Comment:
This is just in case a component reads this getter before the NR data is populated (since, initially, the NR is null).

All components that use this have been updated to handle a falsy (undefined, actually) return value.